app/controllers/api/api_controller.rb

```diff
@@ -8,6 +8,7 @@ class ApiController < ActionController::API
     rescue_from ActionController::ParameterMissing, with: :missing_parameter_response
     rescue_from ActionController::BadRequest, with: :bad_request_response
     rescue_from ActiveRecord::StatementInvalid, with: :bad_request_response
+    rescue_from ArgumentError, with: :bad_request_response
```

Comment:
is this line covered? I ask because I was recently here and have doubts about `Api::ParamErrorFactory` and would rather have coverage

Reply:
When I remove it 
```
          context "when updated_since in an invalid format" do
            it "returns a 400 status" do
              get "/api/v1/participants", params: { filter: { updated_since: "23rm21" } }
              expect(response.status).to eq 400
            end
          end
```
fails

Reply:
I added that spec in the other day (#957) - the issue appeared to be that the `updated_since` was URI encoded and when passed to the query raised the `ActiveRecord::StatementInvalid` error. So I added the line to unencode `updated_since` as well as that `rescue_from`. If things have shifted you might now be raising a different error.